src/main/app/claims/featurePermissionMiddleware.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express'
+
+import { User } from 'idam/user'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+import { Paths as ClaimPaths } from 'claim/paths'
+import * as toBoolean from 'to-boolean'
+import * as config from 'config'
+
+export class FeaturePermissionMiddleware {
+
+  static retrieveUserConsentRole (req: express.Request, res: express.Response, next: express.NextFunction): void {
+    if (!toBoolean(config.get<boolean>('featureToggles.featuresPermission'))) {
```

Comment:
Can you use the `FeatureToggles` class instead of getting direct from config please?